.github/workflows/ci.yml

```diff
@@ -33,6 +33,8 @@ jobs:
           docker-compose up -d && docker-compose down
 
       - name: integration-tests
+        env:
+          CONCOURSE_VERSION: ${{ matrix.concourse-version }}
         run: |
-          sudo make integration-tests-prep-keys
```

Comment:
Tiny ask, but for the sake of quick comprehension could we use the long form of `-E`? `--preserve-env`

Reply:
I decided to just do it.

Reply:
Oh yeah cool sure